eng/Version.Details.xml

```diff
@@ -86,9 +86,9 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>05e49aa98995349ffa26a19020333293ffe99670</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.EditorFeatures" Version="5.0.0-1.25222.12">
-      <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>05e49aa98995349ffa26a19020333293ffe99670</Sha>
+    <Dependency Name="Microsoft.CodeAnalysis.ExternalAccess.Razor.EditorFeatures" Version="5.0.0-1.25227.101">
+      <Uri>https://github.com/dotnet/dotnet</Uri>
+      <Sha>c71a5e287b51dabb97b7e0be5c4ba9b11f840402</Sha>
```

Comment:
This looks wrong, we should update all Roslyn dep versions at once, but this is updating just this singe ExternalAccess dependency.

Reply:
Yes, you should basically not get any updates from dotnet/dotnet in razor since you're still consuming 9.0 versions.
I fixed the excluded assets on the subscription.